docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,54 +81,146 @@ This pattern is necessary because the `flannel` daemon is responsible for settin
 all of the Docker containers created by Kubernetes.  To achieve this, it must run outside of the _main_ Docker daemon.  However,
 it is still useful to use containers for deployment and management, so we create a simpler _bootstrap_ daemon to achieve this.
 
-You can specify k8s version on very node before install:
+## Starting a Cluster
 
-```
+An example cluster is listed below:
+
+| IP Address  |   Role   |
+|-------------|----------|
+|10.10.102.152|   node   |
+|10.10.102.150|node (and master)|
+
+Optional, you can specify version before deployment, otherwise, we'll use latest `hyperkube` release as default.
+
+```sh
 export K8S_VERSION=<your_k8s_version (e.g. 1.0.3)>
 ```
 
-Otherwise, we'll use latest `hyperkube` image as default k8s version.
+Every machine in `NODES` will be deployed as a Node (Minion), and you need to choose one of the `NODES` as Master, see below:
+
+In `cluster/` directory:
+
+```sh
+export NODES="vcap@10.10.102.150 vcap@10.10.102.152"
+export MASTER="vcap@10.10.102.150"
```

Comment:
I'm fully aware of this. 
Yet a "master only" machine is first introduced in the bare-metal ubuntu, which never shows up in the brendan's original idea. 
I believe it's somewhat a design issue, and I love the latter one personally, 'cuz i cannot see a particular reason that we should keep a "master only" machine that rejects pod binding. If the concern comes from resource limitation, we can always use the param for kubelet to limit the pods to be binded.